devel/cctools/Portfile

```diff
@@ -48,6 +56,18 @@ patchfiles              cctools-829-lto.patch \
                         cctools-949-missed-stdio.diff \
                         cctools-949-lipo-segalign-log2fix.diff
 
+
+post-extract {
+    move ${workpath}/cctools-cctools-${version} ${workpath}/cctools-${version}
+    move ${workpath}/ld64-ld64-${ld64_version}  ${workpath}/ld64-${ld64_version}
+}
+
+
+post-extract {
+    move ${workpath}/cctools-cctools-${version} ${workpath}/cctools-${version}
+    move ${workpath}/ld64-ld64-${ld64_version}  ${workpath}/ld64-${ld64_version}
+}
+
```

Comment:
@michaelld seems that the second block was added by mistake.

Reply:
Yup duplicate block ... fixed! Also with the current form of `extract.rename`, need to explicitly set it to `no` so I added that